ibis/pandas/udf.py

```diff
@@ -216,30 +216,47 @@ def pre_execute_analytic_and_reduction_udf(op, *clients, scope=None, **kwargs):
     input_type = op.input_type
     nargs = len(input_type)
 
-    # An execution rule for a simple aggregate node
+    # An execution rule for ungrouped aggregations. Handles:
+    #   1) AggregationNode with no grouping
+    #   2) Some aggregation over a window with no grouping
```

Comment:
What do you mean by "Some aggregation"?

Reply:
Can you please explain exactly which aggregation over a window with no grouping is covered by this?

Reply:
I updated these comments. They were actually inaccurate--these execute rules don't actually handle the Aggregation node. They handle analytic and reduction UDFs.

Reply:
Apparently it _does_ handle Aggregation node as well! Let met update these comments again.